### DISCUSSION TITLE
Codec support?

### DISCUSSION THREAD

**Discussion by tamarw** [mifi/lossless-cut#2253] [Q&A] [answered]
Hi, this might be a really basic question but do I need to download a suite for codec support? I want to simply convert an mp4 to a webm to reduce the filesize. Getting "At least one codec is not supported by the selected output file format. Try another output format or try to disable one or more tracks." I was using other tools for this but I felt like Lossless should be able to do this and I don't need to rely on others, am I wrong?

(Did a search, only found an Issue that reported something completely different, so apologies if I missed something!)

Thanks!

**Comment by mifi** [comment]
if you want to reduce file size you have to either:
- reduce the duration of the video (cut off)
- remove some tracks/streams (e.g. remove a french audio track that you don't need)

other than that you it's not possible convert a file losslessly to reduce the size, unless you re-encode and lose quality. for that, I recommend using a different tool like handbrake, see also https://github.com/mifi/lossless-cut/blob/master/issues.md#faq

**Reply by tamarw** [reply]
Yes, I know, I'm using Handbrake now (that's what I said when I said I was using other tools for this. I'm looking to consolidate if it's possible. And it seems like it should be).

To be clear, I was asking how to reencode (converting a MP4 to a webm should definitely reduce the file size, just like converting a bmp to a jpg to a webp). So I was asking: how do I reencode given that codec error?

**Reply by mifi** [reply, marked as answer]
if you see the faq (i linked above), you can see that it is not possible to re-encode with losslesscut

**Comment by tamarw** [comment]
Gotcha thanks, I was searching for the error message so I didn't stumble upon that. Nonetheless, imo that error message could definitely be clarified or improved for better user understanding.
